GithubConnect/Public/branch.ps1

```diff
@@ -1,4 +1,26 @@
-﻿Function List-GithubBranches {
+Function Get-GithubBranches {

```

Comment:
You reckon this is a better name seeing that there's already a Get-GitHubBranch (singular)?
I know one is supposed to use singular and approved verbs, but I struggled to find a good name for these two functions.